lib/loginprompt.c

```diff
@@ -81,9 +84,8 @@ void login_prompt (char *name, int namesize)
 	 */
 
 	MEMZERO(buf);
-	if (fgets (buf, sizeof buf, stdin) != buf) {
+	if (fgets(buf, sizeof(buf), stdin) == NULL)
```

Comment:
It would be helpful if formatting changes weren't mixed in.

Reply:
I try to keep those to a minimum: only whitespace and (sometimes) parentheses.  And only on hunks where it (subjectively, admittedly) doesn't hurt too much readability (i.e., if I'm doing some huge transformations where I need your attention somewhere else, I'll try to not do it there.

I'll keep in mind your concern.